### ISSUE TITLE
Specifying output Resolution

### ISSUE THREAD

**Issue by Darivo-cmd** [ecmwf/ecmwf-opendata#10]
Hi,
I've installed ecmwf-opendata very rencently. Working on it for a moment without significant issue, I found on ECMWF page that it is possible to mention to the script the output resolution! Wanting to get the HRES (0.1 ~ 9 Km), I tryed to pass on the script that value but failed. URL not found. I've checked the Open data (https://data.ecmwf.int/forecasts/20220608/00z/0p4-beta/oper/), of course,  there is not yet way to request it. 
So my question is :
Does the keyword resol is supported by the server ?
If yes, is there a ways to use it and how, is that means the data will be interpolated or the raw data itself
Thanks in advance for any ansmers

**Comment by floriankrb** [comment]
From the documentation about the ECMWF open data: (https://www.ecmwf.int/en/forecasts/datasets/open-data) _Products are produced at 0.4 degrees resolution in GRIB2 format unless stated otherwise._ 

It would help if you pointed to where you found the information "I found on ECMWF page", what you actually tried "I tryed to pass on the script that value", and what actual error message you got "but failed. URL not found ".

> Does the keyword resol is supported by the server ?
No, the server only supports URLs, i.e. HTTP requests.
